Agents/FenlandTrajectoryAgent/agent/exposure_calculator/templates/absolute_time_filter.sparql

```diff
@@ -0,0 +1,24 @@
+PREFIX fh: <http://www.theworldavatar.com/ontology/OntoFHRS/>
+PREFIX ies4: <http://ies.data.gov.uk/ontology/ies4#>
+PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+PREFIX wgs: <http://www.w3.org/2003/01/geo/wgs84_pos#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+SELECT DISTINCT ?be ?geometry ?wkt ?lat ?long
+WHERE {{
+  BIND("{given_time}"^^xsd:dateTimeStamp AS ?givenTime)
+  BIND(SUBSTR(STR(?givenTime), 1, 4) AS ?timeYear)
+  ?be a fh:BusinessEstablishment ;
+      geo:hasGeometry ?geometry .
+  FILTER(CONTAINS(STR(?geometry), CONCAT("/", ?timeYear)))
```

Comment:
Extracting information from an IRI is not recommended

Reply:
Added a to-do item in commit a8e307a1946c3ba6c332a8811ac5a04730206eeb.